resources/home/dnanexus/utils/generate.py

```
@@ -0,0 +1,280 @@
+
+from .io import make_cnv_session, read_excluded_regions_to_df
+from .query import (
+    get_cnv_file_ids,
+    get_cnv_call_details,
+    find_snv_files,
+    make_url,
+)
+from .util_functions import get_excluded_intervals
+
+
+def gather_snv_data(snv_paths, dx_project) -> dict:
```

Ruff (0.8.0)

Missing type annotation for function argument `snv_paths`

(ANN001)